packages/roosterjs-editor-core/lib/editor/Editor.ts

```diff
@@ -234,7 +234,7 @@ export default class Editor {
     }
 
     // Insert content into editor
-    public insertContent(content: string, option: InsertOption): void {
+    public insertContent(content: string, option?: InsertOption): void {
         if (content) {
             let allNodes = fromHtml(content);
             // If it is to insert on new line, and there are more than one node in the collection, wrap all nodes with
```

Comment:
In line 243
if (option.insertOnNewLine && allNodes.length > 0) {

If the option is null, it will throw exception.

Reply:
good catch!